src/App.js

```diff
@@ -1,102 +1,83 @@
-
 import { useEffect, useState } from 'react';
-import './App.scss';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { faTwitterSquare } from '@fortawesome/free-brands-svg-icons'
 import { faQuoteLeft } from '@fortawesome/free-solid-svg-icons';
-const QuoteUrl='https://gist.githubusercontent.com/camperbot/5a022b72e96c4c9585c32bf6a75f62d9/raw/e3c6895ce42069f0ee7e991229064f167fe8ccdc/quotes.json';
+import './App.scss';
 
+const QuoteUrl = 'https://gist.githubusercontent.com/camperbot/5a022b72e96c4c9585c32bf6a75f62d9/raw/e3c6895ce42069f0ee7e991229064f167fe8ccdc/quotes.json';
+
+const colors = [
+  '#16a085',
+  '#27ae60',
+  '#2c3e50',
+  '#f39c12',
+  '#e74c3c',
+  '#9b59b6',
+  '#FB6964',
+  '#342224',
+  '#472E32',
+  '#BDBB99',
+  '#77B1A9',
+  '#73A857'
+];
 
 function App() {
-  let [randomNumber,setRandomNumber]=useState(0);
-  let [quotes,setQuotes]=useState(null);
-  let [quote,setQuote]=useState(" Life isn’t about getting and having, it’s about giving and being.");
-  let [author,setAuthor]=useState("Kevin Kruse");
-  let [nextIndex,setNextIndex]=useState(0);
-  let colors = [
-    '#16a085',
-    '#27ae60',
-    '#2c3e50',
-    '#f39c12',
-    '#e74c3c',
-    '#9b59b6',
-    '#FB6964',
-    '#342224',
-    '#472E32',
-    '#BDBB99',
-    '#77B1A9',
-    '#73A857'
-  ];
-const fadingEffect=()=>{
-
-  
-  let elements =document.getElementsByClassName("faded");
-
-  Array.from(elements).forEach(element=>{
-    element.classList.add("faded-text");
-  element.addEventListener("animationend",function(){
-    element.classList.remove("faded-text");
-    element.style.opacity="1";
-  });});
-}
+  const [randomNumber, setRandomNumber] = useState(0);
+  const [quotes, setQuotes] = useState(null);
+  const [singleQuote, setSingleQuote] = useState(" Life isn’t about getting and having, it’s about giving and being.");
```

Comment:
should be like this . forgot to clean the string
const [singleQuote, setSingleQuote] = useState(null)
 - you could use only one state as both of author and quote connected.